### ISSUE TITLE
Missing step on how to resize the wsdl vhd

### ISSUE THREAD

**Issue by dangershony** [MicrosoftDocs/WSL#1603]
### Documentation Issue

Step 4 is missing at the end of the step to shot down again wsl 

As explained in this thread
https://github.com/microsoft/WSL/issues/4373#issuecomment-1177323147

### Link to documentation page

https://docs.microsoft.com/en-us/windows/wsl/vhd-size

### Suggested Improvements

I would suggest to add another step at the end of section 4

Shutdown again your wsdl instance `wsl --shutdown`

**Comment by Biswa96** [comment]
`wsl --shutdown` is already mentioned in step 1.

**Comment by dangershony** [comment]
Yes but it needs to be done again at the end of step 4 (see the discussion)
At least I had to do that extra step to make it to work for me.
